### PR TITLE
Version check fix

### PR DESCRIPTION
**Pull request by moollaza**
This ensures that DuckPAN doesn't throw a fatal error when a new version of DuckPAN exists, but the user has the local version pin set lower. Instead it warns that a newer version exists and suggests that you update the local version pin.

Also fixed indentation inconsistencies in DuckPAN.pm and Perl.pm

/cc @zachthompson

### CODE REVIEW

**Comment by zachthompson** [comment]
Do we really have any compelling reason to exit if they're not running the latest DDG/DuckPAN?  I would prefer that, unless we require some minimum version to function properly, upgrading would always just be a recommendation.

File: lib/App/DuckPAN/Perl.pm [hunk 1]
```diff
@@ -131,9 +131,14 @@ sub duckpan_install {
 					$reinstall  = 1;       # Let us roll back, if necessary. Multiple packages may confuse this, but little harm.
 					$install_it = 1;
 				} else {
-					$message    = 'Could not locate version ' . $pin_version . ' of  ' . $package;
-					$install_it = 0;
+					$message = "Could not locate version $pin_version of \'$package\'";
```
Comment (zachthompson) [inline lib/App/DuckPAN/Perl.pm]: Can remove the escapes.

**Comment by moollaza** [comment]
> Do we really have any compelling reason to exit if they're not running the latest DDG/DuckPAN?

Not really? A constant warning that there's a new version would probably be compelling enough. There may be a few edge cases though, such as the recent `Locale::Country` changes you made. If they don't upgrade, the Goodies/Spice's dependent on that code would not work.

**Comment by zachthompson** [comment]
Cool.  Let's remove the exiting in those cases then.  Maybe change emit_error to emit_notice too?  Devs shouldn't have to deal with an app that won't run because of optional versions.

File: lib/App/DuckPAN/Perl.pm (hunk 1 above)
Comment (moollaza) [inline lib/App/DuckPAN/Perl.pm]: :+1:

**Pull request merged**